DesktopModules/Vanjaro/UXManager/Extensions/Menu/GoogleReCaptcha/Controllers/SettingController.cs

```diff
@@ -10,76 +10,55 @@
 using System.Web.Http;
 using Vanjaro.Common.ASPNET.WebAPI;
 using Vanjaro.Common.Engines.UIEngine;
+using Vanjaro.Core;
 using Vanjaro.UXManager.Library.Common;
 
 namespace Vanjaro.UXManager.Extensions.Menu.GoogleReCaptcha.Controllers
 {
-    [ValidateAntiForgeryToken]
-    [AuthorizeAccessRoles(AccessRoles = "admin")]
+    [ValidateAntiForgeryToken]    
```

Comment:
Why did we remove admin access? Now anyone can make a call and get the secret key?

Reply:
changes done